front/src/components/DetailModal/Comment.tsx

```diff
@@ -76,22 +76,31 @@ interface CommentProps {
   toggleEditMode: (text: string) => void;
 }
 
+type mode = 'edit' | 'delete' | 'normal';
```

Comment:
이 부분은 열거형으로 관리해도 좋을 것 같아요
```suggestion
type ModeTypes = 'edit' | 'delete' | 'normal';

enum Mode {
  EDIT = 'edit',
  DELETE = 'delete',
  NORMAL = 'normal',
}
```